pkg/apps/apis/apps/test/ok.go

```diff
@@ -166,6 +166,7 @@ func OkPodTemplate() *kapi.PodTemplateSpec {
 			DNSPolicy:                     kapi.DNSClusterFirst,
 			TerminationGracePeriodSeconds: &one,
 			SchedulerName:                 kapi.DefaultSchedulerName,
+			SecurityContext:               &kapi.PodSecurityContext{},
```

Comment:
@kargakis @deads2k @tnozicka honestly, I have no idea why this is needed.... I spend last hour or so trying to figure out why the TestHandle() is failing, but without this the HasLatestPodTemplate() will see a diff where this field is nil vs. defaulted...

Just want to double check, but the codec i'm passing down to HasLatestPodTemplate is the same as we passed to the original trigger (annotationCodec), so I have no idea what happened there...

Reply:
also let this serve as argument for removing the annotations with encoded deployment configs

Reply:
@mfojtik I am not sure about this, it has been nil even before no matter the codec in tests - it might be sign of something else going wrong

Reply:
@tnozicka it might well be that previous state was wrong :-) i checked the encoded DC in RC and they seems correct (in running server).

Reply:
so my theory is this:

1) Before we haven't decoded the DC nor we did the deepequal in main deployment config controller, because that was part of generic trigger. 
2) The SecurityContext{} is set to empty by default, see kube testing for pod_specs.

So I think this change is fine.